src/lavinmq/vhost.cr

```diff
@@ -113,6 +113,9 @@ module LavinMQ
       headers = properties.headers
       find_all_queues(ex, msg.routing_key, headers, visited, found_queues)
       headers.try(&.delete("BCC"))
+      puts @replicator.min_followers
+      puts @replicator.followers.size
+      return false if @replicator.followers.size < @replicator.min_followers
```

Comment:
Make a method on Replicator, which does the check, so it becomes easier to read, and `@replicationr.followers` duplicates the whole array, which is wasteful if you just want to know its size. 
```suggestion
      return false unless @replicator.has_min_followers?
```